server/controllers/notification.js

```diff
@@ -0,0 +1,52 @@
+const asyncHandler = require("express-async-handler");
+const Notification = require("../models/Notification");
+
+exports.createNotification = asyncHandler(async (req, res) => {
+  const body = req.body;
+  try {
+    //checking if the user has previous notifications
+    const previousNotification = await Notification.findOne({
+      to: body.to,
+      from: req.user.id,
+    });
+
+    if (previousNotification) {
+      previousNotification.notification =
+        previousNotification.notification.concat(body.notification);
+      await previousNotification.save();
+      res.status(200).json(previousNotification);
```

Comment:
Please use status code for resource creation

Ref: https://www.restapitutorial.com/httpstatuscodes.html